CONTRIBUTING.md

```diff
@@ -139,6 +140,20 @@ No matter commit message, or commit content, we do take more emphasis on code re
 
 PR is the only way to make change to Dragonfly project files. To help reviewers better get your purpose, PR description could not be too detailed. We encourage contributors to follow the [PR template](./.github/PULL_REQUEST_TEMPLATE.md) to finish the pull request.
 
+### Developing Environment
+
+During the make contributions to Dragonfly project, some necessary information about the developing environment should be showed to contributors.
```

Comment:
```suggestion
 During the make contributions to Dragonfly project, some necessary information about the developing environment should be showed to contributors.
```
It's weird, may be we can say that like this: 
As a contributor, we should reach an agreement on the version of the tools used in the development environment.  
FYI, WDYT?